metecho/api/admin.py

```diff
@@ -111,11 +112,19 @@ class UserAdmin(admin.ModelAdmin):
     filter_horizontal = ("groups", "user_permissions", "organizations")
 
 
+class GitHubCollaborationInlineAdmin(admin.TabularInline):
+    model = GitHubCollaboration
+
+    nope = lambda *_: False  # noqa: E731
+    has_add_permission = has_delete_permission = has_change_permission = nope
```

Comment:
This inline admin is shown in Projects and GitHub users to easily see which repos a user is a collaborator on. By disabling add/delete/change actions we prevent admins from accidentally modifying this information which comes from the GH API.

The lambda is just a shortcut to avoid defining three methods that just return `False`.